site/frontend/src/pages/compare/compile/table/shortcuts/binary-size-shortcut.vue

```diff
@@ -0,0 +1,53 @@
+<script setup lang="ts">
+import {CompileTestCase} from "../../common";
+import {ArtifactDescription} from "../../../types";
+import Tooltip from "../../../tooltip.vue";
+import {normalizeProfile} from "./utils";
+
+const props = defineProps<{
+  artifact: ArtifactDescription;
+  baseArtifact: ArtifactDescription;
+  testCase: CompileTestCase;
+}>();
+
+function normalizeBackend(backend: string): string {
+  if (backend === "llvm") {
+    return "Llvm";
+  } else if (backend == "cranelift") {
+    return "Cranelift";
+  }
+  return "<invalid backend>";
+}
+</script>
+
+<template>
+  <div class="title">
+    Command for analyzing binary size locally
+    <Tooltip>
+      Execute this command in a checkout of
+      <a href="https://github.com/rust-lang/rustc-perf">rustc-perf</a>, after a
+      `cargo build --release`, to compare binary sizes.
+    </Tooltip>
+  </div>
+
+  <pre><code>./target/release/collector binary_stats \
+    +{{ props.baseArtifact.commit }} \
+    --rustc2 +{{ props.artifact.commit }} \
+    --include {{ testCase.benchmark }} \
+    --profile {{ normalizeProfile(testCase.profile) }} \
+    --backend {{ normalizeBackend(testCase.backend) }}</code></pre>
```

Comment:
Would we want the selectbox with "Diff/Baseline commit/Benchmarked commit" choices here, like for the profiling shortcut?

Reply:
I decided not to add it, to avoid complicating the code. I think that in 99 % you want to see the diff anyway, and getting the binary stats should be much faster than running cachegrind, so I think that it's not worth it to run the command without the diff.

Reply:
Seems fine. And it's also easy to understand once you're not a bigger with rustc-perf that `--rustc2` is the common pattern for diffs.